cli/app.go

```diff
@@ -599,6 +624,74 @@ var app = &cli.App{
 				},
 			},
 		},
+		{
+			Name:            "profiles",
+			Usage:           "Work with CLI profiles",
+			HideHelpCommand: true,
```

Comment:
I don't _think_ you want to hide the help command, right?

Reply:
🤦 was playing with various settings and forgot to remove that. Thanks for the catch, updated!